### ISSUE TITLE
Initialize the project

### ISSUE THREAD

**Issue by torch2424** [AssemblyScript/meta#1]
Thank you very much for making this @dcodeIO :smile: And would appreciate your input on this @MaxGraey 

Again, please let me know if I am "stepping on any toes" during this process, but hoping this all goes well :smile: 

For the first issue, I think it'd be good to define what this repo is for! I like the idea of calling it "meta", because it's kind of like a project about the project haha! :+1: 

Anyways, (to me) this is inspired by the following:

https://github.com/vuejs/roadmap
https://github.com/ampproject/meta
https://github.com/JSFoundation/standards
https://opensource.guide/leadership-and-governance/

In other words, a place to open issues, and discuss the project in general, and ideas on how to help support it. Thus, I think it'd be good to initialize the repo with a following:

* README.md
* CODE_OF_CONDUCT

Which we may already have in the main project repo. I would not mind doing this, but not sure if I have access :thinking: But yeah, let me know what you think, and we can kick this off (o discuss more if you don't mind) :+1:

**Comment by MaxGraey** [comment]
I think it will be great move to this repo all media like logos with in different formats for easy search wdyt?

**Comment by torch2424** [comment]
@MaxGraey I agree with that :smile: Seems like a fitting use of the repo!

**Comment by dcodeIO** [comment]
Agreed, should move everything meta to this repo. Perhaps also CONTRIBUTING.md. All participants in this issue have full access to the meta repo, and suggesting to do the removals from other places as PRs. Some of the documents are referenced in the sidebar docs and the wiki afaik (maybe just replace these with a link to the meta repo).

**Comment by MaxGraey** [comment]
How about RFCs for changes? Yes, we follow TS and ECMAScript as close as possible but sometimes we need implement some things little bit different or discuss about some minor extensions. So I don't think we need separate repo for this and probably much better manage add this suggestions in current meta repo? wdyt?

**Comment by torch2424** [comment]
@dcodeIO 

> Agreed, should move everything meta to this repo. Perhaps also CONTRIBUTING.md. All participants in this issue have full access to the meta repo, and suggesting to do the removals from other places as PRs.

Awesome, I'll submit some of the documents now :smile: 

One question, for the `CONTRIBUTING.md` would that cover the main `AssemblyScript/assemblyscript` repo? Or all repos in the org as well?

>  Some of the documents are referenced in the sidebar docs and the wiki afaik (maybe just replace these with a link to the meta repo).

By the way, I think you did an amazing job on the docs website :tada: :smile: I do think we should try to have a single place for this stuff. But I'm open to having the docs point to the meta repo, or vice versa.

@MaxGraey 

> How about RFCs for changes? Yes, we follow TS and ECMAScript as close as possible but sometimes we need implement some things little bit different or discuss about some minor extensions. So I don't think we need separate repo for this and probably much better manage add this suggestions in current meta repo? wdyt?

I'm definitely open to this / think it is a good idea! I'm very flexible on what goes into the "meta" repo :smile: 

**Lastly,**

Should I add a code of conduct as well? I don't feel like we need one right now, but those types of things I feel like are better handled sooner rather than later. :smile:

**Comment by dcodeIO** [comment]
> One question, for the CONTRIBUTING.md would that cover the main AssemblyScript/assemblyscript repo? 

Hmm, now that you mention it, the file is somewhat specific to the compiler repo, yeah. Let's leave it there.

> I do think we should try to have a single place for this stuff. But

Does this mean that you'd prefer to add the meta information to the docs directly?

> Should I add a code of conduct as well?

There's [this one](https://github.com/AssemblyScript/assemblyscript/blob/master/CODE_OF_CONDUCT.md) on the compiler repo currently, but feel free to improve :)

**Comment by torch2424** [comment]
> Hmm, now that you mention it, the file is somewhat specific to the compiler repo, yeah. Let's leave it there.

Awesome, I'll open an issue for creating a general contributing doc :smile: 

> Does this mean that you'd prefer to add the meta information to the docs directly?

Oops yeah I was a bit unclear there. I agree, things like markdown files that were originally on the wiki. Super open to moving them to this repo, and simply changing the link on the docs :smile: 

> There's this one on the compiler repo currently, but feel free to improve :)

Oh that's perfect! I'll add it to my PR :smile:
